x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/configuration_step/configuration_step_form.tsx

```diff
@@ -443,6 +458,43 @@ export const ConfigurationStepForm: FC<CreateAnalyticsStepProps> = ({
         loadingItems={loadingFieldOptions}
         setFormState={setFormState}
       />
+      {includes.length >= 2 && (
+        <>
+          <EuiFormRow
+            data-test-subj="mlAnalyticsCreateJobWizardScatterplotMatrixFormRow"
+            label={i18n.translate('xpack.ml.dataframe.analytics.create.scatterplotMatrixLabel', {
+              defaultMessage: 'Scatterplot matrix',
+            })}
+            helpText={i18n.translate(
+              'xpack.ml.dataframe.analytics.create.scatterplotMatrixLabelHelpText',
+              {
+                defaultMessage:
+                  'Visualizes the relationships between pairs of selected included fields',
+              }
+            )}
+            fullWidth
+          >
+            <Fragment />
```

Comment:
I presume you don't need this Fragment 
```suggestion
```

Reply:
EUI doesn't allow to use the component without a child, I copied the whole code block from another part and will leave it like this for UI consistency for now.